### ISSUE TITLE
Script editor completion doesn't suggest members of a script after `:`

### ISSUE THREAD

**Issue by me2beats** [godotengine/godot#47720]
**Godot version:**

3.2.3

**OS/device including version:**

Linux

**Issue description:**

![completion](https://user-images.githubusercontent.com/16458555/114000538-8b8f8880-9874-11eb-90ec-fcfd32713bc4.png)

![completion1](https://user-images.githubusercontent.com/16458555/114000572-921e0000-9874-11eb-8ea3-2ec75a728211.png)

**Steps to reproduce:**
1. Create a script `new_script.gd`:
```gdscript
class X:
	pass

```
2. Create another script with this code
```gdscript
const scr = preload("res://new_script.gd")
func _ready():
	var x: scr.
```
As you can see the completion doesn't appear
It should suggest `X`.

**Minimal reproduction project:**

**Comment by akien-mga** [comment]
This seems partially fixed in `3.x`, as the completion does show up for me.
It shows invalid results though, with the actual inner class at the bottom of the list.

So #48037 might still be needed and fix that.

The original issue is still reproducible in the current `master` branch (e2a37295f52366ab04cbb87ecbd945dd23de9517). CC @Chaosus

**Comment by akien-mga** [comment]
This is now fixed in 3.6 by #48037, but as pointed out it's still reproducible for 4.0.
